packages/endpoint/src/SnapshotInterface.ts

```diff
@@ -20,5 +20,5 @@ export default interface SnapshotInterface {
     ...args: readonly [...Parameters<E['key']>] | readonly [null]
   ) => ErrorTypes | undefined;
 
-  readonly fetchStart: number;
+  readonly fetchedAt: number;
```

Comment:
does this make it a breaking change?

Reply:
this was only released in the previous beta. we don't promise api stability of new features in betas.